.template/lib/template/errors.rb

```diff
@@ -1,21 +1 @@
-module Template
-  class Errors
-    delegate :empty?, to: :errors
-
-    def initialize
-      @errors = []
-    end
-
-    def add(error_message)
-      errors << error_message
-    end
-
-    def to_s
-      errors.join("#{'-' * 80}\n")
-    end
-
-    private
-
-    attr_reader :errors
-  end
-end
+class Template::Errors < Template::Messages; end
```

Comment:
I am not sure about this architecture implementation as it feels a bit unnatural/strange 🤔  

Just like flashes, how about instead having one class/object e.g. `Template::Messages` and when adding a new message to the list, one has to add a type e.g. {type: :info, text: "Please set up ..."}, {type: :error, text: "Please set up ..."}. Then when looping for messages, the messages will be filtered by type. 

Note that there could be some helper method like Template::Messages.add_info, Template::Messages.add_error (this could be created automatically for each type using [define_method](https://stackoverflow.com/questions/38233060/define-method-how-to-dynamically-create-methods-with-arguments)

Reply:
Let me rework the message system in another PR. I need to revisit all the places that we add the message and also deal with how to print out all messages of specific type 🙏

Reply:
I'll be working on it when doing [Issue: Refactor Template Helpers](https://github.com/nimblehq/rails-templates/issues/321)